metricbeat/docs/modules/linux.asciidoc

```diff
@@ -0,0 +1,49 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+[[metricbeat-module-linux]]
+== linux module
+
+beta[]
+
+The Linux module reports on metrics exclusive to the Linux kernel and GNU/Linux OS.
+
+[float]
+=== Example configuration
+
+The linux module supports the standard configuration options that are described
+in <<configuration-metricbeat>>. Here is an example configuration:
+
+[source,yaml]
+----
+metricbeat.modules:
+- module: linux
+  period: 10s
+  metricsets:
+    - "pageinfo"
+    # - ksm
+    # - conntrack
+    # - perf
```

Comment:
I see the `perf` metricset is gone now. Looks like references to it, like this one here, need to be removed from configs, docs, etc.

Reply:
I knew I was missing something!